api/tests/unit/domain/usecases/get-campaign-participation-result_test.js

```diff
@@ -6,13 +6,15 @@ const { UserNotAuthorizedToAccessEntity } = require('../../../../lib/domain/erro
 describe('Unit | UseCase | get-campaign-participation-result', () => {
 
   const campaignParticipationId = 1;
+  const assessmentId = 4;
   const userId = 2;
   const campaignId = 'campaignId';
   const otherUserId = 3;
   const campaignParticipationResult = { id: 'foo' };
   const campaignParticipation = {
     campaignId,
     userId,
+    assessment: { id: assessmentId }
```

Comment:
Il n'y a plus de propriété `assessment` dans `CampaignParticipation` donc :

```suggestion
    assessmentId
```

Après si le test passe même avec cette mauvaise propriété `assessment` il y a de quoi se demander si cette ligne est utile…

Reply:
Effectivement, c'était inutile ! Bien vu 👍